<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QWidget" name="side_bar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);</string>
   </property>
   <widget class="QPushButton" name="patient_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;

border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Patients</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/patientdownload .png</normaloff>../Medico_modern/patientdownload .png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="dash_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
border-bottom-color: rgb(0, 0, 0);
selection-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/dashboard.png</normaloff>../Medico_modern/dashboard.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="appointment_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Appointments</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/appointmentdownload (1).png</normaloff>../Medico_modern/appointmentdownload (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="dentist_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Dentists</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/dentistimages.png</normaloff>../Medico_modern/dentistimages.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="expense_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Expense</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/billingdownload (1).png</normaloff>../Medico_modern/billingdownload (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="payment_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Payments</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/billingdownload (1).png</normaloff>../Medico_modern/billingdownload (1).png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="add_fname">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="add_fname_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_lname">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="add_lname_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_phn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="add_phn_edit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_address">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="add_address_edit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_occu">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Occupation</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="add_occu_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_email">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="add_email_edit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_ref">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Reference</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="add_ref_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_depart">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>If from abroad, date of departure</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="add_depart_edit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_complain">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>981</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Chief Complain</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="add_complain_edit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>981</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>525</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
</string>
   </property>
   <property name="text">
    <string>Proceed To Medical History</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_age">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="add_age_edit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_sex">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Sex</string>
   </property>
  </widget>
  <widget class="QComboBox" name="add_sex_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#add_sex_edit,
#add_sex_edit QAbstractItemView {
    background-color: white;
}
</string>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Others</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="return_btn">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>525</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
</string>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>1041</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/daniel-frank-wKbWAMlHgNo-unsplash.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>530</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>side_bar</zorder>
  <zorder>add_fname</zorder>
  <zorder>add_fname_edit</zorder>
  <zorder>add_lname</zorder>
  <zorder>add_lname_edit</zorder>
  <zorder>add_phn</zorder>
  <zorder>add_phn_edit</zorder>
  <zorder>add_address</zorder>
  <zorder>add_address_edit</zorder>
  <zorder>add_occu</zorder>
  <zorder>add_occu_edit</zorder>
  <zorder>add_email</zorder>
  <zorder>add_email_edit</zorder>
  <zorder>add_ref</zorder>
  <zorder>add_ref_edit</zorder>
  <zorder>add_depart</zorder>
  <zorder>add_depart_edit</zorder>
  <zorder>add_complain</zorder>
  <zorder>add_complain_edit</zorder>
  <zorder>add_age</zorder>
  <zorder>add_age_edit</zorder>
  <zorder>add_sex</zorder>
  <zorder>add_sex_edit</zorder>
  <zorder>label_3</zorder>
  <zorder>next_btn</zorder>
  <zorder>label_4</zorder>
  <zorder>return_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
